### PR TITLE
Hotfix/single instance

### PR DESCRIPTION
**Pull request by jkutianski**
Allow just 1 instance of the app

### CODE REVIEW

File: src/main/python/main.py [hunk 1]
```diff
@@ -21,6 +21,9 @@
 # http://timlehr.com/python-exception-hooks-with-qt-message-box/
 from util import init_logger
 
+#Single instance app
+from single_application import QtSingleApplication
```
Comment (xyzz) [inline src/main/python/main.py:25]: Please move the import below (right above the `app = QtSingleApplication` line) so it is not imported when using linux-recorder

**Review by xyzz — CHANGES_REQUESTED** [review]
It doesn't seem to work on Linux AppImage for me - I can run two instances just fine.

What OS did you check on?

**Comment by jkutianski** [comment]
> It doesn't seem to work on Linux AppImage for me - I can run two instances just fine.
> 
> What OS did you check on?

Tried on Windows and now on Linux. And I had the same problem on Linux. Very strange.
This code is based on this comment from stackoverflow
https://stackoverflow.com/questions/12712360/qtsingleapplication-for-pyside-or-pyqt/12712362#12712362

File: src/main/python/main.py (hunk 1 above)
Comment (jkutianski) [reply src/main/python/main.py]: Done

**Comment by xyzz** [comment]
I think the current implementation should call activateWindow so that it pops up the existing application window

it's ok if it only works on windows for now, we can fix it later

**Pull request closed without merging**